Content.Client/Communications/UI/CommunicationsConsoleBoundUserInterface.cs

```diff
@@ -59,8 +55,22 @@ public void EmergencyShuttleButtonPressed()
 
         public void AnnounceButtonPressed(string message)
         {
-            var msg = message.Length <= 256 ? message.Trim() : $"{message.Trim().Substring(0, 256)}...";
-            SendMessage(new CommunicationsConsoleAnnounceMessage(msg));
+            var msg = (message.Length <= 256 ? message.Trim() : $"{message.Trim().Substring(0, 256)}...").ToCharArray();
+
+            // No more than 2 newlines, other replaced to spaces
```

Comment:
This should be checked on the server as well, when it receives the CommunicationsConsoleAnnounceMessage (same for the 256 char limit)